frontend/src/components/shared/ComparisonMatrix.vue

```diff
@@ -3,29 +3,32 @@
     <table class="table">
       <thead>
         <tr>
-          <th class="has-nowrap"></th>
-          <th class="has-nowrap" v-for="cn in columnNames" :key="cn">{{ cn }}</th>
+          <th class="has-nowrap px-3 py-3"></th>
+          <th v-for="cn in columnNames" :key="cn" class="has-nowrap">{{ cn }}</th>
         </tr>
       </thead>
       <tbody>
         <tr v-for="(rn, i) in rowNames" :key="rn + i">
           <th class="has-nowrap">{{ rn }}</th>
-          <td v-for="(cn, j) in columnNames" :key="cn + j" :class="{selected: isSelectedCell(i, j)}">
+          <td v-for="(cn, j) in columnNames"
```

Comment:
It looks better with the previous code. 
Old one: 

<img width="469" alt="Screenshot 2021-09-22 at 09 48 33" src="https://user-images.githubusercontent.com/5833722/134303731-ac850490-f66e-419c-974e-16813d05359e.png">

New one: 
<img width="454" alt="Screenshot 2021-09-22 at 09 48 40" src="https://user-images.githubusercontent.com/5833722/134303738-ef9e2e57-90bf-4b69-ab9e-3738e338b749.png">

Reply:
Great that you spotted this! It should be back to normal via 53766814
<img width="416" alt="image" src="https://user-images.githubusercontent.com/23480589/134469086-ba04aac6-3b8e-4e02-8f55-21f2d28e5f07.png">